installation/install

```diff
@@ -85,11 +85,13 @@ function handle_error
 
 function get_os_type
 {
-	which zypper &> /dev/null && os="opensuse"
 	which pacman &> /dev/null && os="archlinux"
-	which dnf &> /dev/null && os="fedora"
 	which apt-get &> /dev/null && os="debian"
+	which apt-get &> /dev/null && os="deepin"
```

Comment:
deepin seems to be a debian based system, it should be handled as such, and no line should be added for it
`which apt-get &> /dev/null && os="debian"
+	which apt-get &> /dev/null && os="deepin"`
if you have an Ubuntu, Mint, Debian (etc..) it will choose deepin, but they are debian based systems ;)